api/src/core/sql/map/mv_words_phrases_languages-v8.sql

```diff
@@ -0,0 +1,78 @@
+-- drop old view
+
+drop view if exists v_map_words_and_phrases_with_tr_info;
+
+-- mv_words_languages definition
+
+DROP materialized view if exists mv_words_languages;
```

Comment:
please put all materialized view code into the schema update file.

Reply:
and remember to update the final.schema.sql file with any changes so we can quickly glance at the current schema.